cmd/skopeo/copy.go

```diff
@@ -231,11 +233,17 @@ func (opts *copyOptions) run(args []string, stdout io.Writer) (retErr error) {
 		return err
 	}
 
+	identity, err := reference.ParseNamed(opts.signIdentity)
```

Comment:
This fails if `opts.signIdentity` is `""`; so this parsing needs to be conditional on that.